testing/regress/ecl/dropzone_query_test.ecl

```diff
@@ -22,3 +22,5 @@ import Std.File AS FileServices;
 output(NOTHOR(FileServices.GetDefaultDropZone()));
 
 output(NOTHOR(FileServices.GetDropZones()));
+
+output(NOTHOR(FileServices.GetLandingZones()));
```

Comment:
Should there be a corresponding key file change?

Reply:
@richardkchapman no, because this test is marked as not being compared against a key file.   It would probably be better if it did, but that would require consistent environments.  I checked it did generate the correct output.